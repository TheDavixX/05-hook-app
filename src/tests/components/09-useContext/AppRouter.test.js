import React from 'react';
import { mount } from 'enzyme';
import { AppRouter } from '../../../components/09-useContext/AppRouter';
import { UserContext } from '../../../components/09-useContext/UserContext';

describe('Pruebas en <LoginScreen />', ()=>{

    const user = {
        name: 'David',
        email: 'davidestevez1000@gmail.com'
    };

    const wrapper = mount( 
        <UserContext.Provider value={{
            user
        }}>
            <AppRouter /> 
        </UserContext.Provider>
    );

    test('debe mostrarse correctamente', () => {
       
        expect( wrapper ).toMatchSnapshot();
        
    });
    

});