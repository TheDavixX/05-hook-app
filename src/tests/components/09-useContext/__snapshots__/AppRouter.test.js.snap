// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <LoginScreen /> debe mostrarse correctamente 1`] = `
<div>
  <nav
    className="navbar navbar-expand-sm navbar-dark bg-dark"
  >
    <div
      className="container-fluid"
    >
      <a
        aria-current={null}
        className="navbar-brand"
        href="/home"
        onClick={[Function]}
      >
        useContext
      </a>
      <div
        className="collapse navbar-collapse"
        id="navbarNavAltMarkup"
      >
        <div
          className="navbar-nav"
        >
          <a
            aria-current={null}
            className="nav-item nav-link"
            href="/home"
            onClick={[Function]}
          >
             Home 
          </a>
          <a
            aria-current={null}
            className="nav-item nav-link"
            href="/about"
            onClick={[Function]}
          >
             About 
          </a>
          <a
            aria-current={null}
            className="nav-item nav-link"
            href="/login"
            onClick={[Function]}
          >
             Login 
          </a>
        </div>
      </div>
    </div>
  </nav>
  <div
    className="container"
  >
    Array [
      <h1>
        Home Screen
      </h1>,
      <hr />,
      <pre>
        {
   "name": "David",
   "email": "davidestevez1000@gmail.com"
}
      </pre>,
    ]
  </div>
</div>
`;
